### PR TITLE
[draco] Update to v1.5.6

### PR DESCRIPTION
**Pull request by cbrl**
This PR updates Draco to version 1.5.6.

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by cbrl** [comment]
There might be some problems introduced in the changes to Draco's CMake files. The failing checks all report that PDAL failed to compile because the Draco lib couldn't be opened, though the library does exist.

Interestingly, I had tested this change by compiling Assimp, which depends on Draco, but it appears that the Assimp port doesn't actually enable the Draco-dependent features since it'll compile just fine even if I delete `draco.lib`.

**Comment by cbrl** [comment]
https://github.com/microsoft/vcpkg/pull/31220/commits/b17c4ef10e6f93353fb81047c7188480e1359548 seems to fix the problem, but I don't know if it's actually a good solution. The change from using `CMAKE_INSTALL_FULL_LIBDIR` to `CMAKE_INSTALL_LIBDIR` in the pkgconfig file seems to have caused this issue, though I'm not entirely sure what the reason for that is.

**Comment by Cheney-W** [comment]
Regarding the [disable-symlinks.patch](https://github.com/microsoft/vcpkg/pull/31220/files#diff-d0d304b50a1e896ffc1c999e0a68600c9631a856b848655f2575286fe3291815) file, please do not modify the line endings of the file.

File: ports/draco/portfile.cmake [hunk 1]
```diff
@@ -3,12 +3,13 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO google/draco
-    REF 1.5.3
-    SHA512 8575ea78e0d8025facddbd42453b0251387f4e31eb0854135e050fc26aaf0d28ed30ccc3f93578fdc6cdb50369c2ef735291f1f5fb60238b289e0ee019446e1d
+    REF 1.5.6
```
Comment (Cheney-W) [inline ports/draco/portfile.cmake:6]:
```suggestion
    REF "${VERSION}"
```

**Comment by cbrl** [comment]
Sorry about that. The line endings are fixed now, and the portfile uses the `VERSION` variable instead of the hardcoded ref value.

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**